internal/jujuapi/api.go

```diff
@@ -36,17 +36,24 @@ func APIHandler(ctx context.Context, jimm *jimm.JIMM, p Params) http.Handler {
 // ModelHandler creates an http.Handler for "/model" endpoints.
 func ModelHandler(ctx context.Context, jimm *jimm.JIMM, p Params) http.Handler {
 	mux := http.NewServeMux()
-	mux.Handle("/{uuid}/api", &jimmhttp.WSHandler{
+	mux.Handle("/model/{uuid}/api", http.StripPrefix("/model", &jimmhttp.WSHandler{
```

Comment:
Why are you including the `/model/` here? Previously I removed it from here and just did stripPrefix outside of this function.

Reply:
the reason is that we want to proxy the req with the model in its path, and it doesn't make sense to strip it outside and then add it inside the proxy HTTP function.